src/fiatconnect/saga.ts

```diff
@@ -277,14 +280,57 @@ export function* handleSelectFiatConnectQuote({
   payload: params,
 }: ReturnType<typeof selectFiatConnectQuote>) {
   const { quote } = params
+
```

Comment:
it seems like this saga does something much different on quote selection when kyc is required vs when it is not. However, after the user has gone through kyc, it seems as though we'll want the other stuff to still happen. Do you have a plan for that?

Reply:
seems like a naive approach would be to create a new saga that is "everything after kyc" and invoke that (somehow) when the user has completed submitting their kyc info. 

Another (still fairly naive) would be to simulate selecting the quote again once the user has submitted kyc info

Reply:
Hm, you mean after they've completed the Persona flow in particular, we would want to "resume" with the remainder of the saga (basically run the whole saga as it was before this PR) as normal? Simulating selecting the quote again (just dispatching the same action we do when a user selects a quote) isn't the worst idea imo, it's the most "robust" in that it actually ensures that the KYC states stored in IHL are as we expect, rather than just assuming they are after completing Persona KYC.

Also, after sending KYC to Persona, we'll need to hit POST /kyc for the provider they've selected; rather than doing that separately as an ad-hoc thing, that can just be done when calling this saga again, since the logic to do that already exists there. If we go this route, perhaps the naming of this saga could use an update, though.